tools/toolchain/install_cp2k_toolchain.sh

```diff
@@ -1069,8 +1149,8 @@ gen_arch_file() {
 #
 CXX         = CC
 CXXFLAGS    = \${CXXFLAGS} -I\\\${CUDA_PATH}/include -std=c++11
-GPUVER      = K20X
-NVCC        = \${NVCC} -D__GNUC__=4 -D__GNUC_MINOR__=9 -Xcompiler=--std=gnu++98
+GPUVER      = \${GPUVER}
+NVCC        = \${NVCC} -D__GNUC__=4 -D__GNUC_MINOR__=9
```

Comment:
please note: for older compilers you have to pass `-std=c++11' to NVCC as well, see https://github.com/cp2k/dbcsr/commit/3069161ac6d40bb2f0728f5e97464d1ca0cc669b

Reply:
I pass `--std=c++11` to NVCC via the variable `NVFLAGS`(cf line 1097 of the same file)

Reply:
sorry, didn't check further down

Reply:
no problem :) sorry for the late reply